Source/items.cpp

```diff
@@ -3441,6 +3443,45 @@ bool DoOil(Player &player, int cii)
 
 [[nodiscard]] StringOrView PrintItemPower(char plidx, const Item &item)
 {
+	int param1, param2 = 0;

```

Comment:
why only assign 0 to param2? param1 is still uninitialized

Reply:
Mistake